src/evidently/collector/config.py

```diff
@@ -55,6 +55,16 @@ def is_ready(self, config: "CollectorConfig", storage: "CollectorStorage") -> bo
         return False
 
 
+class RowsCountTrigger(CollectorTrigger):
+    rows_count: int = 1
+
+    def is_ready(self, config: "CollectorConfig", storage: "CollectorStorage") -> bool:
+        buffer_size = storage.get_buffer_size(config.id)
+        if buffer_size > 0 and buffer_size >= self.rows_count:
```

Comment:
You can just return the result of this check instead of literals

Reply:
I'm not sure if I understand what you are pointing to, can you explain? Is it that you wouldn't return literal True/False, but instead the integer difference between buffer_size and rows_count or sth. else? 🤔

Reply:
just `return buffer_size > 0 and buffer_size >= self.rows_count`

Reply:
alright, adjusted in 9cbbe9ecd5f62ae94d4dcece22cc7ed01c781f3e